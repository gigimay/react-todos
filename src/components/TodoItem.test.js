import React from 'react';
import TodoItem from './TodoItem';
import renderer from 'react-test-renderer';

it('TodoItem renders correctly', () => {
  const tree = renderer.create(
    <TodoItem label="Item" />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});

it('TodoItem renders correctly', () => {
  const tree = renderer.create(
    <TodoItem label="Item" complete={true}/>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
