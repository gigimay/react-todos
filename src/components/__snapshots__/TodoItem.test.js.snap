exports[`test TodoItem renders correctly 1`] = `
<li
  className=""
  onClick={undefined}>
  Item
</li>
`;

exports[`test TodoItem renders correctly 2`] = `
<li
  className="complete"
  onClick={undefined}>
  Item
</li>
`;
